### ISSUE TITLE
Event "CategoryWatch" in CategoryModel is broken

### ISSUE THREAD

**Issue by R-J** [vanilla/vanilla#1914]
Trying to hook the event with `public function CategoryModel_CategoryWatch_Handler($Sender)` doesn't work, because the event is fired that way:

~~~
public static function CategoryWatch() {
...
      Gdn::PluginManager()->EventArguments['CategoryIDs'] =& $Watch;
      Gdn::PluginManager()->FireEvent('CategoryWatch');
~~~

I searched in the forum and found that discussion: http://vanillaforums.org/discussion/comment/173148#Comment_173148
Seems like the error hasn't been reported back then.

**Comment by bleistivt** [comment]
Looks like this was fixed in #3740

**Comment by R-J** [comment]
Yes, it is fixed.
